packages/search-input/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+
+{
+  "extends": "../../package.tsconfig.json",
+  "compilerOptions": {
+    "declarationDir": "dist",
+    "outDir": "dist",
+    "rootDir": "src"
+  },
+  "include": ["src/**/*"],
+  "references": [
+    {
+      "path": "../lib"
```

Comment:
You're missing a few packages

Reply:
Thanks for catching it! I always forget this has to be manual.

Reply:
actually i think Adam added a scrip to add these

Reply:
`check:deps`

Reply:
Hm that script didn't work for me. I updated the file manually anyway!